### ISSUE TITLE
CI: docs build failing

### ISSUE THREAD

**Issue by victorlin** [nextstrain/ncov#1126]
First noticed in [CI run](https://github.com/nextstrain/ncov/actions/runs/9979143141/job/27577642985):

```
▶ Run make html
Running Sphinx v5.3.0

Extension error:
Could not import extension sphinxarg.ext (exception: cannot import name 'get_default_settings' from 'docutils.frontend' (/usr/share/miniconda/envs/test/lib/python3.12/site-packages/docutils/frontend.py))
make: *** [Makefile:20: html] Error 2
Error: Process completed with exit code 2.
```

**Comment by joverlee521** [comment]
Ah, same issue as https://github.com/nextstrain/docs.nextstrain.org/issues/214. 

I should have realized this would be an issue elsewhere, we should probably pin all uses of sphinx-argparse. [GH search](https://github.com/search?q=org%3Anextstrain%20sphinx-argparse&type=code) shows we need to update ncov, auspice, and nextclade

**Comment by joverlee521** [comment]
Oh actually, [upstream issue](https://github.com/sphinx-doc/sphinx-argparse/issues/57) seems to be resolved in new release.

**Comment by joverlee521** [comment]
Ah, [fix in sphinx-argparse](https://github.com/sphinx-doc/sphinx-argparse/commit/a1621bc8b4abbd598df82038b67d9ae3f479308b) requires docutils>=0.19 but we pin docutils<0.18.

**Comment by joverlee521** [comment]
Unpinning docutils seems to fix the docs build: https://github.com/nextstrain/ncov/pull/1127

**Comment by victorlin** [comment]
Fixed by #1127
